lua/telescope/command.lua

```diff
@@ -101,4 +102,32 @@ function command.run_command(args)
   end
 end
 
+function command.load_command(cmd,...)
+  local args = {...}
+  local user_opts = {}
+  user_opts['cmd'] = cmd
+  user_opts.opts = {}
+
+  for _,arg in ipairs(args) do
+    if arg:find('=',1) == nil then
+      user_opts['extension_type'] = arg
+    else
+      local param = utils.split_by_pattern(arg,'[^=]+')
+      if param[1] == 'theme' then
+        user_opts['theme'] = param[2]
+      elseif param[2]:find(',',1) ~= nil then
+        if param[1] == 'search' then
```

Comment:
What about other parameters which could expect a `,`. I would do the other way around. Define a table with options that need to be converted like `find_command`, `vimgrep_arguments`, `selections` and convert them.

We need other opinions on that
And i gotta go now :)

Still thanks for your work :)

Reply:
you mean create a table that collection table type keywords then if match we convert the value right? But check issue some extensions developer will create their own keywords .so these keywords we don't know which type it is. so I try to use comma as a deal to convert value to table

Reply:
As i said earlier these extension dev gotta do something like this to support lua and vim side:
```lua
opts.option = type(opts.option) == 'table' and opts.option or vim.split(opts.option, ',')
```
I kinda have it that way not convert one rather than converting and its not expected :|

Reply:
If they can do this. everything come to easy :)